src/libsyntax/parse/parser.rs

```diff
@@ -3003,8 +3016,7 @@ impl<'a> Parser<'a> {
                         this.parse_assoc_expr_with(op.precedence() + 1,
                             LhsExpr::NotYetParsed)
                 }),
-                // the only operator handled here is `...` (the other non-associative operators are
-                // special-cased above)
+                // no operators are currently handled here
```

Comment:
AFAIK this code is dead. [Here is the comment](https://github.com/rust-lang/rust/blob/stable/src/libsyntax/parse/parser.rs#L2836-L2837) from before my original #30884. I made use of that "future convenience" since I originally didn't care to parse `a...`, but to produce a good error I had to back that out and use the same special case as `a..`.